net/net-tools/Makefile

```diff
@@ -68,5 +84,11 @@ define Package/net-tools-route/install
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/route $(1)/usr/libexec/net-tools-route
 endef
 
+define Package/net-tools-netstat/install
```

Comment:
And here - sort it too

Reply:
Do you plan to address this comment?

Reply:
@mcbridematt Do you plan to address the comment?

Reply:
Sorry, I've been busy on other projects. I'll try and address this in the next few days